docs/data/material/getting-started/templates/marketing-page/components/Features.js

```diff
@@ -42,7 +42,7 @@ const items = [
 const Chip = styled(MuiChip)(({ theme }) => ({
   variants: [
     {
-      props: ({ selected }) => selected,
+      props: ({ selected }) => !!selected,
```

Comment:
Have to fix this because the type of `selected` is `boolean | undefined`.

Should we loosen the type like of props? to be `() => boolean | null | undefined`?

Reply:
I don't mind the `!!` to cast into boolean

Reply:
Does this mean this is a breaking change?

Reply:
I would argue it's a fix as it was always supposed to be `() => boolean` (it's only `boolean` in the `CSSObjectWithVariants` type), but it wasn't actually because the type wasn't being correctly picked up.

It's one of those fixes that could be perceived as a breaking change.

Maybe we could compromise with a minor release and an explanation on the changelog?

Reply:
This would be the last thing that we need to decide on. Similar to the `styles` now being required, this is a fix that might be perceived as breaking.

My proposal is that we release it with in a minor version and a note.

Reply:
>My proposal is that we release it with in a minor version and a note.

Let's do that.